src/styles/sidebar/components/menu.scss

```diff
@@ -11,8 +11,7 @@
 
 // Toggle button that opens the menu.
 .menu__toggle {
-  @include buttons.button--icon-only;
-
+  @include buttons.button--icon-only(true, inherit);
```

Comment:
Ah! One other thing I just thought of. If you use param names here, it makes two improvements:

1. You don't have to pass a value for `$with-active-state` and
2. It's easier to human parse what's going on here

e.g.

`@include buttons.button--icon-only($coarse-min-size: inherit);`

Reply:
Yep, I like that better